testing/jormungandr-integration-tests/build.rs

```diff
@@ -14,5 +14,5 @@ fn main() {
     let jormungandr_name = option_env!("JORMUNGANDR_NAME").unwrap_or("jormungandr");
     println!("cargo:rustc-env=JOR_CLI_NAME={}", jor_cli_name);
     println!("cargo:rustc-env=JORMUNGANDR_NAME={}", jormungandr_name);
-    println!("cargo:rustc-env=RUST_BACKTRACE={}", "full");
+    println!("cargo:rustc-env=RUST_BACKTRACE=full");
```

Comment:
It's better to remove this. It should be up to the users (or the CI environment configuration) how they want to have their backtraces.